components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -40,9 +56,30 @@ class CustomTabsToolbarFeature(
         }
     }
 
+    internal fun addCloseButton(bitmap: Bitmap?) {
+        val drawableIcon = bitmap?.let { icon ->
+            BitmapDrawable(context.resources, icon).also {
+                if (it.isSmallerThan(MAX_CLOSE_BUTTON_SIZE_DP)) return@also
+            }
+        } ?: ContextCompat.getDrawable(context, R.drawable.mozac_ic_close)
+
+        drawableIcon?.let {
+            val button = Toolbar.ActionButton(
+                it, context.getString(R.string.mozac_feature_customtabs_exit_button)
+            ) { closeListener.invoke() }
+            toolbar.addNavigationAction(button)
+        }
+    }
+
     override fun stop() {}
 
+    internal fun Drawable.isSmallerThan(maxSize: Int): Boolean = with(context.resources) {
+        pxToDp(minimumWidth) <= maxSize && pxToDp(minimumHeight) <= maxSize
+    }
```

Comment:
Let's move those into `support-ktx`.

Reply:
I considered this as well, but I'd need to pass the resources into it as well and I thought that didn't make a great extension function to be public so I had it internal i.e. `Drawable.isSmallerThan(Int)` seems better than `Drawable.isSmallerThan(Resources, Int)`

If we don't mind that, then sure!

Reply:
Maybe instead of converting the values of `Drawable` you could convert `maxSize` to what Drawable uses when passing it in?

Reply:
I could, but I would still need the Resources/DisplayMetrics to know the screen density to make that calculation. I'll try to think of a better way to do this... 🤔